rds/example/main.tf

```diff
@@ -0,0 +1,9 @@
+module "test1" {
```

Comment:
Could name these something like simplest case instead of test1. I think we will want to have more examples listed here as we go forward